src/Feature/Layout/Feature_Layout.re

```diff
@@ -0,0 +1,284 @@
+open Utility;
+
+type direction =
+  | Up
+  | Left
+  | Down
+  | Right;
+
+type t('content) =
+  | Split([ | `Horizontal | `Vertical], list(t('content)))
+  | Window({
+      weight: float,
+      content: 'content,
+    })
+  | Empty;
+
+[@deriving show({with_path: false})]
+type window('content) = {
+  content: 'content,
+  x: int,
+  y: int,
+  width: int,
+  height: int,
+};
```

Comment:
This name isn't great, as it's specifically a window as a product of `layout`, so more like `laidOutWindow`? But preferably using a less awkward term.

Reply:
Maybe `measuredWindow` or `sizedWindow`?

Reply:
I like `sizedWindow`, so going with that. Thanks!